test/Tests/src/Data/Json_Spec.enso

```diff
@@ -164,6 +164,13 @@ spec =
             obj1 . should_not_equal <| JS_Object.from_pairs [["a", 43], ["b", 123]]
             obj1 . should_not_equal <| JS_Object.from_pairs [["a", 42], ["b", JS_Object.from_pairs [["c",1], ["d",3]]]]
 
+        Test.specify "should be able to set values" <|
```

Comment:
One additional test here - as we are using the JS set value it would be good to check the original is not mutated.

Reply:
Done